documentation/src/main/asciidoc/topics/proc_configuring_remotecm_bean.adoc

```diff
@@ -1,36 +1,42 @@
 [id='configuring-remotecm-bean_{context}']
 = Setting Up the RemoteCacheManager
+Configure your application to use remote caches on {brandname} clusters.
 
-. Provide the location for the {brandname} server so the starter can create the `RemoteCacheManager` bean.
+. Provide the location for {brandname} Server so the starter can create the `RemoteCacheManager` bean.
```

Comment:
I'm not sure about the "location for {brandname} Server" wording, first because "location" is so generic it doesn't tell the reader anything, and second because we should advise users to include more than one server in the initial list.

Maybe "Provide the listening addresses of one or more {brandname} Servers" would sound better?